qdrant-landing/content/documentation/fastembed/fastembed-semantic-search.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "FastEmbed & Qdrant"
+weight: 3
+---
+
+# Using FastEmbed with Qdrant for Vector Search
+
+## Install Qdrant Client
+The Qdrant Python Client has a simple in-memory mode that lets you try semantic search locally. 
+```python
+pip install qdrant-client
+```
```

Comment:
not sure that this statement is required here, maybe the comment we have below (when we instantiate QdrantClient) is enough